src/sync/gll.ts

```diff
@@ -0,0 +1,42 @@
+import PQueue from 'queue-promise';
+import { ILogger } from '../types';
+import { GllTaskError } from '../errors';
+
+let GLL = new PQueue({ concurrent: 5 });
+
+export function setConcurrency(value: number) {
+  GLL = new PQueue({ concurrent: value });
```

Comment:
If this method is called during the node execution, then we could have two instances running simultaneously? I feel it is a bug.

Reply:
You're right, removed this method since there is no way to safely update the concurrency for now.

Reply:
Ok. Thanks!